### PR TITLE
Fix prerequisite installation scripts and instructions for latest drake and add CI

### LINKED ISSUE #2: update to latest drake install instructions
@jamiesnape kindly offered to update this repo as done in https://github.com/RussTedrake/underactuated/pull/234

### PR DESCRIPTION
**Pull request by jamiesnape**
Closes #2.

CI needs GitHub Actions to be enabled on this account: https://github.com/features/actions/signup/

`planar_iiwa_viz_test.py` is disabled since it fails to find the needed URDF file.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "htmlbook"]
 	path = htmlbook
 	url = https://github.com/RussTedrake/htmlbook
+[submodule "underactuated"]
```
Comment (RussTedrake) [inline .gitmodules]: PlanarSceneGraphVisualizer is in drake master now.  i was hoping that I wouldn't need to include underactuated here?  I will test.

Comment (RussTedrake) [reply .gitmodules]: looks like it is (only?) blocking on https://github.com/RobotLocomotion/drake/pull/12349 . i think we should get this PR in without underactuated (or trimesh, etc) and just don't add the planar_iiwa test to CI for now.  

btw -- thank you so much for doing all this!

**Comment by jamiesnape** [comment]
BTW You are missing a license file. Would you like me to add BSD-3-Clause to the root?

**Comment by RussTedrake** [comment]
Yes, thanks!

File: .gitmodules (hunk 1 above)
Comment (jamiesnape) [reply .gitmodules]: Done.

**Comment by jamiesnape** [comment]
Done.

**Comment by RussTedrake** [comment]
LGTM.  Thanks Jamie!

**Pull request merged**